.github/workflows/build.yml

```diff
@@ -35,20 +35,19 @@ jobs:
         working-directory: src/github.com/kubernetes-sigs/cri-tools
 
   releases:
-    name: releases
+    name: releases / on ${{ matrix.os }}
     runs-on: ${{ matrix.os }}
-    timeout-minutes: 10
+    timeout-minutes: 15
```

Comment:
how close was it that we had to go to 15?

Reply:
Raised to `20` to be sure. It was happening intermittently to Mac OS release build.